File: Array/Missing_Number.py
def missing_number(nums):
    result = len(nums)
    for i in range(len(nums)-1):
        result += i - nums[i]
    return result


missing_number([0, 9, 8, 3, 5, 4, 7, 1, 3, 2])  # missing_number = 6

File: Binary/missingNumber.c
int main() {
  
  int nums[] = {0, 1, 2, 3, 5, 6, 7, 8, 9};
  int len = sizeof(nums) / sizeof(nums[0]);
  int result = 0;
  
  for (int i = 0; i < len; i++){
    result = result ^ (i+1) ^ nums[i]; // result = 4   
  }

  return 0;
}

